test/ssl_test_ctx.h

```diff
@@ -214,6 +214,10 @@ typedef struct {
     STACK_OF(X509_NAME) *expected_client_ca_names;
     /* Whether to use SCTP for the transport */
     int use_sctp;
+    /* Enable SSL_MODE_DTLS_SCTP_LABEL_LENGTH_BUG on client side*/
```

Comment:
`... side */`

Reply:
Fixed in https://github.com/openssl/openssl/pull/7957/commits/47b846eb2e30e4d0e4e02c0f1c529a54d4874a13.

Thanks for reporting.